cc2538-bsl.py

```diff
@@ -1221,7 +1221,7 @@ def cli_setup():
                 raise Exception("NO CRC32 match: Local = 0x%x, "
                                 "Target = 0x%x" % (crc_local, crc_target))
 
-        if args.ieee_address != 0:
+        if args.ieee_address != 0 and args.ieee_address != None:
```

Comment:
```suggestion
        if args.ieee_address is not None:
```
The `args.ieee_address` attribute will never be 0 as the argparser argument will only be a string or `None`. Even if `-i0` is provided as an argument, it will come in as `"0"`. (Unless I'm missing a line that converts the integer, but I don't see anything else touching `ieee_address`)

Then when checking if something is (or is not) `None`, using `is (not) None` is the usual way.